### ISSUE TITLE
Increased Descriptor selection

### ISSUE THREAD

**Issue by michaelmacisaac** [openkim/kliff#77]
Hi!
Based on section 3.3 of 'Kliff a ... interatomic potentials,' adding more descriptors is being explored. I am wondering what descriptors are being considered? I am specifically wanting to explore the use of SOAP.

**Comment by mjwen** [comment]
Yes, we are exploring the possibility to support more descriptors. @ipcamit has been working on a general support of the descriptors in the [DScribe](https://github.com/SINGROUP/dscribe) package (SOAP included).

**Comment by mjwen** [comment]
We now have a `libdescriptor` library for this: https://github.com/openkim/libdescriptor
